elyra/pipeline/registry.py

```diff
@@ -523,6 +523,17 @@ def parse_component_properties(self, component_body, component_path):
 
                     # Add to existing parameter list
                     component_parameters['parameters'].append({"id": new_parameter_info['parameter_ref']})
+                    if(default_value is not None and len(default_value) > 0):
```

Comment:
This could probably be simplified to: `if default_value:`.  If `False` or `0`, it should do the right thing, but that would be where the issue might lie.  But if its going to be a string regardless, then its save to simplify the conditional.